templates/.eslintignore.tmpl

```diff
@@ -4,3 +4,6 @@ jest.config.js
 .eslintrc.js
 cypress
 cypress.config.js
+src/common/chains/**
+src/common/eips/**
+src/common/hardforks/**
```

Comment:
Ignoring all converted json files for eslint errors as we want no to minimum changes for now in these files ( for avoiding any breaking functionality and easy maintenance of these in near future by updating these json with released eth/comon json files)